code/style.css

```diff
@@ -1,7 +1,356 @@
+    
+  }
+  .picture {
+    width: auto;
+    margin: 1rem;
+  }
+  
+  .picture img {
+    width: 400px;
+  }
+  
+  .information {
+    margin: 1rem;
+    padding: 1rem;
+    min-width: 400px;
+    border: 2px solid white;
+  
+  }
+  
+  /*HERO*/
+  .hero {
+    height: 100vh;
+    width: 100%;
+    background: white;
+    display: flex;
+    align-items: center;
+  }
+  
+  .hero video {
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 5;
+    object-fit: cover;
+  }
+  
+  .caption {
+    position: sticky;
+    z-index: 20;
+    background:rgba(255,255,255, 0.5);
+    color: #5D463D;
+    width: 100%;
+    height: 150px;
+    padding: 20px;
+    display: flex;
+    justify-items: center;
+    flex-direction: column;
+  }
+  .caption h1 {
+    font-family: 'Oswald', Sans-Serif;
+    text-transform: uppercase;
+    font-size: 40px;
+    margin: 5px;
+  }
+  .caption h2 {
+    font-family: 'Quattrocento', Serif;
+    font-size: 20px;
+    margin: 0px;
+  
+  
+  }
+  
+  /*HEADER*/
+  .header {
+    padding: 20px;
+    height: 30px;
+    background-color:#90A8AD;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  .sticky {
+    position: fixed;
+    top: 0;
+    width: 100%;
+  }
+  .hamburger {
+    display: flex;
+    height: 25px;
+    width: 25px;
+    flex-direction: column;
+    justify-content: space-between;
+  
+  }
+  
+  .hamburger div {
+    height: 4px;
+    width: 25px;
+    background: whitesmoke;
+  
+  }
+  
+  nav {
+     display: none; 
+  }
+  
+  nav a {
+    text-decoration: none;
+    color: whitesmoke;
+    margin: 15px;
+  }
+  nav a:hover {
+    border-bottom: 1.5px solid white;
+  }
+  
+  
+  
+  /*ACCORDION*/
+  
+  h3 {
+    text-align: center;
+  }
+  .accordion {
+    width: 80%;
+    max-width: 800px;
+    margin: 2rem auto;
+    font-family: 'Quattrocento', Serif;
+  
+  }
+  .accordion-item {
+    background-color: white;
+    color: #5D463D;
+    margin: 1rem;
+    border-radius: 0.5rem;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.5)
+  }
+  .question {
+    padding:0.5rem 1rem;
+    min-height: 3rem;
+    font-weight: bold;
+    display: flex;
+    align-items: center;
+    position: relative;
+    cursor: pointer;
+  }
+  .question::after {
+    content: "\25BE";
+    font-size: 1.5rem;
+    position: absolute;
+    right: 1rem;
+    transition: transform 0.2s ease-in-out;
+  }
+  .question.active::after {
+    /* content: "\2212";  */
+    transform: rotate(-180deg);
+  }
+  .answer {
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.2s ease-out;
+  }
+  
+  .answer p {
+    padding: 1rem;
+    background-color: whitesmoke;
+    border-top: 3px solid;
+    border-image: linear-gradient(to right, transparent, #5D463D,#EBD0C7) 1;
+  }
+  
+  /* FORM */
+  
+  input:valid {
+  /* What does that look like */
+  
+  }
+  input:invalid {
+  /*What does that look like*/
+  }
```

Comment:
Try to remove unused code, or maybe you wanna go back and check this out later on?